docs/shared/using-nx/affected.md

```diff
@@ -1,27 +1,122 @@
-# How Affected Works
+# Run Only Tasks Affected by a PR
 
-{% callout type="note" title="First thing!" %}
-Before reading this guide, [check out the mental model guide](/concepts/mental-model). It will help you understand how computation caching fits into the rest of Nx.
-{% /callout %}
+s
+When you run `nx test app1`, you are telling Nx to run the `app1:test` task plus all the tasks it depends on.
```

Comment:
Do we really need these first set of paragraphs? I would almost just start with "As you workspace grows,..." and then mentioning maybe "retesting, rebuilding, re-linting..." basically emphasize it is not just for testing.

Then right after "How does this work?", I'd place in a terminal window showing `nx affected -t <task>` or an alternate syntax. Basically to have the actual command right there. And then proceed with the explanations